### PR TITLE
Docs/apply feedback suggestions

### PR DESCRIPTION
**Pull request by dariodjuric**
Solves the following points from the feedback:

- I really enjoyed how the express kit by @TapaiBalazs documented the .env file. I would like that here too.
- If you pass the -d flag to docker compose up, it'll run the process in the background and not lock up the user's terminal. I recommend doing this instead.
- Running the docs command I only get 1 file. Is this intentional?
- I'd like to see default routes in the README so a user knows how to interact quickly with this codebase.

### CODE REVIEW

File: starters/deno-oak-denodb/README.md [hunk 1]
```diff
@@ -192,6 +192,35 @@ This starter kit has the following structure in the src folder:
...
+### Default API routes
+
+The starter kit comes with a REST API for the healthcheck and a GraphQL API for everything else.
+
+The REST API's healtcheck is available at http://localhost:3333/health by default and returns the status of the databases.
+
+The GraphQL API is available at http://localhost:3333/graphql by default. If you execute a GET on it, you will get the GraphQL playground (in non-production only). If you execute a POST, you can use the GraphQL API. The API has the following operations:
+
+- `getTechnologies` gets you all technologies in the database
+- `getTechnology` gets you a single technology from the database
```
Comment (vyktoremario) [inline starters/deno-oak-denodb/README.md:204]: Changed to `technologies` and `technology`

File: starters/deno-oak-denodb/README.md [hunk 2]
```diff
@@ -192,6 +192,35 @@ This starter kit has the following structure in the src folder:
...
+```
+{
+  "id": "34a3a076-31ec-4138-972d-41db0e8ec654",
+  "displayName": "Oak",
+  "description": "A middleware framework for Deno's native HTTP server, Deno Deploy and Node.js 16.5 and later. It also includes a middleware router.",
+  "url": "https://oakserver.github.io/oak/",
+  "createdAt": "2022-12-10T17:32:41.086Z",
+  "updatedAt": "2022-12-10T17:32:41.086Z"
+}
+```
```
Comment (vyktoremario) [inline starters/deno-oak-denodb/README.md:220]:
```suggestion
```json
{
  "id": "34a3a076-31ec-4138-972d-41db0e8ec654",
  "displayName": "Oak",
  "description": "A middleware framework for Deno's native HTTP server, Deno Deploy and Node.js 16.5 and later. It also includes a middleware router.",
  "url": "https://oakserver.github.io/oak/",
  "createdAt": "2022-12-10T17:32:41.086Z",
  "updatedAt": "2022-12-10T17:32:41.086Z"
}
```

**Review by vyktoremario — CHANGES_REQUESTED** [review]
Looks 👍 . A few comment.
And you have a merge conflict

File: starters/deno-oak-denodb/README.md (hunk 1 above)
Comment (dariodjuric) [reply starters/deno-oak-denodb/README.md:204]: Ah, I was working on an old version of the branch. Thanks.

File: starters/deno-oak-denodb/README.md (hunk 2 above)
Comment (dariodjuric) [reply starters/deno-oak-denodb/README.md:220]: Good idea, thanks.

**Comment by dariodjuric** [comment]
> Do we need Table of content on the readme like other kits?

Couldn't hurt! Adding it now.

**Pull request merged**